client/views/room/Header/Omnichannel/QuickActions/QuickActions.tsx

```diff
@@ -162,22 +163,26 @@ const QuickActions = ({ room, className }: { room: IRoom; className: BoxProps['c
 		openModal(id);
 	});
 
+	const isSubscribedToRoom = !!subscription;
+
+	const omnichannelRouteConfig = useOmnichannelRouteConfig();
+
 	const hasManagerRole = useRole('livechat-manager');
 
-	const roomOpen = room && room.open && ((room.servedBy && room.servedBy._id === uid) || hasManagerRole);
+	const roomOpen = room && room.open && ((room.u && room.u._id === uid) || hasManagerRole);
```

Comment:
Why are you changing this field? Omnichannel rooms don't rely on `u` property, when it comes to Omnichannel rooms, `servedBy` is the right field to determine who's serving the chat.

Reply:
The room object over here doesn't have `servedBy` property. So I think that it has been transformed to `room.u` property

Reply:
It might be an issue with the Room Provider that's not providing the field we need here.
If so, we need to make sure the field is present in the room object rather than using another property.

Reply:
@renatobecker I've done this change, could you please review it again? Thanks :smile: